### ISSUE TITLE
Circuit rebooter something

### ISSUE THREAD

**Issue by Time-Green** [yogstation13/yogstation#2059]
How is the consensus on allowing engineers to fix emagged airlocks and APC's by some easier method, like being able to remove the circuit directly and fix/replace it? So that they dont have to completely rebuild it, and considering how easy it is to emag.

**Comment by Slinky5890** [comment]
Yeah i'm fine with that. Though personally I always use RCDs cos i'm a lazy bastard

**Comment by Time-Green** [comment]
Who doesn't?

**Comment by doerakpoes** [comment]
I don't

**Comment by Mrcreepssss** [comment]
yes, please yes.
